base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,45 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+	WCHAR szShell[256];
+	WCHAR szPath[MAX_PATH];
+    LPWSTR szExplorer = NULL;
+	DWORD dwType;
+	DWORD dwBufferSize = sizeof(szShell);
+    BOOL bIsSystemShell = TRUE;
+
+    if (!GetModuleFileName(NULL, szPath, MAX_PATH))
```

Comment:
Avoiding hardcoding the buffer size:
```suggestion
    if (!GetModuleFileNameW(NULL, szPath, _countof(szPath)))
```

Reply:
Is _countof recommended here or would ARRAYSIZE() be better? I've seen ARRAYSIZE used in other programs and someone once recommended it to me.

Reply:
ARRAYSIZE is the common one yes but ROS uses _countof (why, I don't know).

Reply:
I'd say, it's better use the one that is already used in this module - in case of explorer it's `_countof`, however there's one occurrence of `_ARRAYSIZE`, which could have been added by mistake.